aten/src/ATen/TensorImpl.h

```diff
@@ -185,22 +229,27 @@ struct AT_API TensorImpl : public c10::intrusive_ptr_target {
     return n;
   }
   bool compute_contiguous() const;
-
- protected:
   void refresh_numel() {
```

Comment:
why are these no longer protected?

Reply:
They are moved to `private` above, which is even more restricted :D